md/bonita-docker-installation.md

```diff
@@ -61,10 +80,62 @@ See the official PostgreSQL documentation for more details.
 docker run --name bonita_postgres --link mydbpostgres:postgres -d -p 8080:8080 bonitasoft/bonita-subscription:${varVersion}.0
 ```
 
-#### PostgreSQL
-if you don't want to run your database in a docker container, some things need to be configured separately: 
+#### Using docker-compose
+
+Create a file stack.yml with the following content:
+```
+# Use tech_user/secret as user/password credentials
+version: '3'
+
+services:
+  db:
+    image: postgres:11.6
+    environment:
+      POSTGRES_PASSWORD: example
+    restart: always
+    command:
+      - -c
+      - max_prepared_transactions=100
```

Comment:
Ok, there is a simple way to set max_prepared_transactions.
So why not, line 76, also set it directly from the docker run command, (`docker run --name mydbpostgres -v ~/Documents/Docker/Volumes/custom_postgres/:/docker-entrypoint-initdb.d -e POSTGRES_PASSWORD=mysecretpassword -d postgres:11.2 -c 'max_prepared_transactions=100'`) instead of creating a shell script?

Reply:
Done

Reply:
:+1: